tests/scripts/thread-cert/Cert_5_1_01_RouterAttach.py

```diff
@@ -168,6 +179,317 @@ def test(self):
+        #             - Mode TLV
+        #             - Response TLV
+        #             - Timeout TLV
+        #             - TLV Request TLV
+        #             - Address16 TLV
+        #             - Network Data TLV
+        #             - Route64 TLV (optional)
+        #             - Version TLV
+        #             - MLE Frame Counter TLV (optional)
+        #         The following TLV MUST NOT be present in the MLE Child ID Request:
+        #             - Address Registration TLV
+
+        _pkt = pkts.filter_wpan_src64(ROUTER).\
+            filter_wpan_dst64(LEADER).\
+            filter_mle_cmd(MLE_CHILD_ID_REQUEST).\
+            filter(lambda p: {\
+                              LINK_LAYER_FRAME_COUNTER_TLV,\
+                              MODE_TLV,\
+                              RESPONSE_TLV,\
+                              TIMEOUT_TLV,\
+                              TLV_REQUEST_TLV,\
+                              ADDRESS16_TLV,\
+                              NETWORK_DATA_TLV,\
+                              ROUTE64_TLV,\
+                              VERSION_TLV,\
+                              MLE_FRAME_COUNTER_TLV\
+                              } <= set(p.mle.tlv.type) or\
+                             {\
+                              LINK_LAYER_FRAME_COUNTER_TLV,\
+                              MODE_TLV,\
+                              RESPONSE_TLV,\
+                              TIMEOUT_TLV,\
+                              TLV_REQUEST_TLV,\
+                              ADDRESS16_TLV,\
+                              NETWORK_DATA_TLV,\
+                              VERSION_TLV\
+                               } <= set(p.mle.tlv.type)).\
+                   must_next()
+        _pkt.must_not_verify(lambda p: (ADDRESS_REGISTRATION_TLV) in p.mle.tlv.type)
+
+        # Step 5: Leader responds with a Child ID Response.
+        #         The following TLVs MUST be present in the Child ID Response:
+        #             - Address16 TLV
+        #             - Leader Data TLV
+        #             - Network Data TLV
+        #             - Source Address TLV
+        #             - Route64 TLV (if requested)
+
+        pkts.filter_wpan_src64(LEADER).\
+            filter_wpan_dst64(ROUTER).\
+            filter_mle_cmd(MLE_CHILD_ID_RESPONSE).\
+            filter(lambda p: {\
+                              ADDRESS16_TLV,\
+                              LEADER_DATA_TLV,\
+                              NETWORK_DATA_TLV,\
+                              SOURCE_ADDRESS_TLV,\
+                              ROUTE64_TLV\
+                              } <= set(p.mle.tlv.type) or\
+                             {\
+                              ADDRESS16_TLV,\
+                              LEADER_DATA_TLV,\
+                              NETWORK_DATA_TLV,\
+                              SOURCE_ADDRESS_TLV\
+                               } <= set(p.mle.tlv.type)).\
```

Comment:
```suggestion
            filter(lambda p: {\
                              ADDRESS16_TLV,\
                              LEADER_DATA_TLV,\
                              NETWORK_DATA_TLV,\
                              SOURCE_ADDRESS_TLV\
                               } <= set(p.mle.tlv.type)).\
```
I think these two are same.